tests/migrations/test_loader.py

```diff
@@ -381,6 +381,19 @@ def test_check_consistent_history(self):
         with self.assertRaisesMessage(InconsistentMigrationHistory, msg):
             loader.check_consistent_history(connection)
 
+    @override_settings(
+        MIGRATION_MODULES={'migrations': 'migrations.test_migrations_squashed_extra'},
+        INSTALLED_APPS=['migrations'],
+    )
+    def test_check_consistent_history_squashed(self):
+        "Tests that check_consistent_history doesn't misbehave with squashed migrations."
```

Comment:
State the expected behavior: MigrationLoader.check_consistent_history() should ignore unapplied squashed migrations that have all of their `replaces` applied.